### PR TITLE
Remove elastic 6.x support

### PR DESCRIPTION
**Pull request by pavolloffay**
Related to https://github.com/jaegertracing/jaeger/issues/637

### CODE REVIEW

**Comment by jpkrohling** [comment]
And related to jaegertracing/jaeger#665

File: content/docs/deployment.md [hunk 1]
```diff
@@ -193,7 +193,7 @@ usercert = ~/.cassandra/client-cert
 
 ### ElasticSearch
 
-Supported versions: 5.x, 6.x
```
Comment (jpkrohling) [inline content/docs/deployment.md]: As people have asked explicitly about 6.x before, I'd add a note saying that "We are aware of problems with 6.x and would welcome contributions in fixing them", linking to https://github.com/jaegertracing/jaeger/issues/665

**Comment by pavolloffay** [comment]
If this gets merged I will submit another to fix https://github.com/jaegertracing/jaeger/tree/master/plugin/storage/es#elasticsearch-support

File: content/docs/deployment.md (hunk 1 above)
Comment (pavolloffay) [reply content/docs/deployment.md]: I would rather point them to the https://github.com/jaegertracing/jaeger/issues/637. Which can track various es6 issues

**Comment by yurishkuro** [comment]
-1

I am not clear how the issue with indices is related to 6.x support

**Comment by pavolloffay** [comment]
I will close this per https://github.com/jaegertracing/jaeger/pull/921 and https://github.com/jaegertracing/jaeger/issues/665#issuecomment-403405429

**Pull request closed without merging**